gradle.properties

```diff
@@ -21,6 +21,7 @@ org.gradle.jvmargs=-XX:MaxPermSize=2048m -XX:+CMSClassUnloadingEnabled -XX:+Heap
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 org.gradle.parallel=true
+org.gradle.java.home=C:\\Program Files\\Java\\jdk1.8.0_161
```

Comment:
Why this change?

Reply:
Else the gradlew check was failing.
I searched the reason for error and did the change.
Should I revert it?